src/ipfs.io/index.html

```diff
@@ -42,7 +42,7 @@
 [Interplanetary Networks](//ipn.io) and many contributors
 from the open source community.
 
-<iframe id="stars" src="http://ghbtns.com/github-btn.html?user=ipfs&repo=ipfs&type=watch&count=true&size=large" allowtransparency="true" frameborder="0" scrolling="0" width="152px" height="30px"></iframe>
```

Comment:
maybe this should be `src="//ghbtns.com...` to preserve whatever scheme its coming from? (not sure if there will be warnings about this